### PR TITLE
Allow generating validation rules with boolean or array args.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: src/Shell/Task/ModelTask.php [hunk 1]
```diff
@@ -732,11 +732,11 @@ public function fieldValidation($schema, $fieldName, array $metaData, $primaryKe
         }
 
         if (in_array($fieldName, (array)$primaryKey)) {
-            $rules['allowEmpty'] = ['create'];
+            $rules['allowEmpty'] = ["'create'"];
         } elseif ($metaData['null'] === true) {
             $rules['allowEmpty'] = [];
         } else {
-            $rules['requirePresence'] = ['create'];
+            $rules['requirePresence'] = ["'create'"];
```
Comment (markstory) [inline src/Shell/Task/ModelTask.php]: Could this potentially result in working templates no longer working as they require quotes now?

Comment (ADmad) [reply src/Shell/Task/ModelTask.php]: Yes it potentially can if someone has overridden this method.

So we can document this change and bump bake version accordingly.

Comment (markstory) [reply src/Shell/Task/ModelTask.php]: We'd have to go to 1.6 I think.

Comment (ADmad) [reply src/Shell/Task/ModelTask.php]: Yup, fine by me.

**Pull request merged**